src/WebUI/locales/en.yml

```diff
@@ -502,13 +503,31 @@ character:
         notify:
           warning: A large shield cannot be used on horseback
       upgrade:
-        upgradesTitle: Item Upgrades
+        upgradesTitle: "Item upgrades"
+        tooltip:
+          title: "Upgrade item"
+          description: "You can upgrade your item for a heirloom point"
+        validation:
+          maxRank: "The item already has a maximum rank"
```

Comment:
Recommend change to -> "This item has already been upgraded to the highest point possible"